src/Symfony/Component/AssetMapper/Compressor/BrotliCompressor.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\AssetMapper\Compressor;
+
+use Symfony\Component\Process\ExecutableFinder;
+use Symfony\Component\Process\Process;
+
+/**
+ * Compresses a file using Brotli.
+ *
+ * @author Kévin Dunglas <kevin@dunglas.dev>
+ */
+final class BrotliCompressor implements CompressorInterface
+{
+    public function __construct(
+        private ?string $executable = null,
+    ) {
+    }
+
+    public function compress(string $path): void
+    {
+        if (file_exists("$path.br")) {
```

Comment:
Should we skip if it already exist ? Are we deleting the compressed file when we write a new version of the uncompressed file ?

Reply:
Indeed we shouldn't skip but overwrite, it's safer. I'll update the code.